draft-ietf-quic-http.md

```diff
@@ -1343,13 +1334,12 @@ parameter consists of a setting identifier and a value, both encoded as QUIC
 variable-length integers.
 
 ~~~~~~~~~~~~~~~  drawing
- 0                   1                   2                   3
- 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                        Identifier (i)                       ...
-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                           Value (i)                         ...
-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+SETTINGS Frame {
+  Type (i) = 0x4,
+  Length (i),
+  Identifier (i),
+  Value (i),
+}
```

Comment:
This is wrong, sorry. The text discusses that the frame contains 0 or more parameters and the old drawing presents the format of the parameter, not the frame. I think what you want to do is present  the pair of (Identifier, Value) with cardinality, and then update the fig title to sat "SETTINGS Frame".